Tweet-311-Denver/tweet_311_denver_ui
# Adding Tweet component

## What does this PR do?
- Adds the basic layout for the `Tweet` component 
- Adds styling for `Tweet` component 
- Adds some error handling for the `Tweet` component 
- Adds two images to the `assests` folder

## How to test?
- run `git fetch`
- git checkout `tweetcomponent-gi`
- run `npm i` 
- `expo start`
- You should be able to see the changes 

## Issues:
- This PR closes no issues but we will be able to check off some to-do boxes on the appropriate card

## Screenshots:
![Screen Shot 2020-02-17 at 4 23 18 PM](https://user-images.githubusercontent.com/48968224/74691858-d0082280-51a1-11ea-8e10-6431c54b58ca.png)

## Notes:
** ************Important************ **
- I made no changes to `App` so this will not work right away, we should be able to merge in your PR and then this one. Just make sure and give this a good look over before a merge and I can make the needed corrections.
 
Thanks!
@cammac60
